6_visualize/src/prep_outro_funs.R

```diff
@@ -1,7 +1,15 @@
 prep_outro_rdgs_fun <- function(rdg_ind="1_fetch/out/rapid_dep_sites.rds.ind", gage_col_config, outro_placement, legend_text_cfg, opacity=1) {
   # project from lat/lon to plot coordinates. . this properly belongs in a 2_process step, but we're releasing tomorrow
   proj_str <- "+proj=lcc +lat_1=34.83333333333334 +lat_2=32.5 +lat_0=31.83333333333333 +lon_0=-81 +x_0=609600 +y_0=0 +ellps=GRS80 +units=m +no_defs "
-  rdgs <- readRDS(sc_retrieve(rdg_ind)) %>%
+
+  rdgs <- readRDS(sc_retrieve(rdg_ind))
```

Comment:
New pattern we just started trying in `delaware-model-prep` is to put all getter targets in a separate, disconnected remake file called "getters.yml". For now, at least, that would also mean adding a `remake_file='getters.yml'` argument to every call to `sc_retrieve` throughout the project. (But if you wanted to start by making a PR in scipiper to add a default `remake_file='getters.yml'` to `sc_retrieve` you could save some typing in this repo).
  This may not be a change you want to make in this PR, but it has saved us numerous unnecessary rebuilds in that other project and I think it would be a noticeable improvement for vizstorm projects, too.

Reply:
Could you explain what you mean by `getter` targets? If this is something that has proven useful to stop rebuilds in other vizstorm projects, then I think it makes sense here. We have plans to do some overhaul on this pipeline but that isn't the highest priority, so I think it would be best to capture these ideas in an issue and revisit later.

Reply:
Oh, sorry, I did mean to elaborate on that "getter" thing - I meant targets whose command is `gd_get`, `s3_get`, etc. See https://github.com/USGS-R/scipiper/issues/70#issuecomment-643671102. The requisite overhaul for implementing this involves changing a lot of files, but it's pretty much just a search-and-edit operation, so it was refreshingly fast to do for `delaware-model-prep`.